### DISCUSSION TITLE
Clarification on SAF Pipeline and Maximum Streamline Length

### DISCUSSION THREAD

**Discussion by solaire678** [dmitrishastin/SAF#5] [Q&A]
Dear Professor Shastin,

I have a question regarding the SAF pipeline. Would it still be suitable if the maximum streamline length is increased to 85 mm (instead of the 40 mm used in the manuscript)? Are there any additional considerations I should take into account? For example, would it be necessary to increase the number of streamlines beyond 5 million?

Thank you for your time and insights.

Best regards,
Cristóbal

**Comment by dmitrishastin** [comment]
Hi Cristóbal,

Thank you for taking interest in our work. 

From memory, increasing streamline length is likely to pick up deeper white matter fibres - just because of how they are arranged. The pipeline should still work, although it may take longer to run (more filtering required). I would therefore try with a smaller number of streamlines (100-500K) to test then increase as you feel appropriate. Ultimately, the number of streamlines required is a balance between what is needed for your particular application, and resource consumption. 

Hope this helps

Dmitri

**Comment by solaire678** [comment]
Dear Professor Shastin,

Thank you very much for your reply :)

Best regards,

Cristóbal
